index.html

```diff
@@ -39,7 +39,6 @@
       }
       return null;
     };
-
```

Comment:
This might seem silly, but can we keep this space? It's there to separate the functions from the constants. :P